lib/ts/recipe/dashboard/api/usersGet.ts

```diff
@@ -47,7 +47,12 @@ export type Response = {
     }[];
 };
 
-export default async function usersGet(_: APIInterface, options: APIOptions, userContext: any): Promise<Response> {
```

Comment:
this API's Response type should also contain tenantIds: string[]

Reply:
Done